ntbs-service/Pages/Notifications/Edit/Episode.cshtml.cs

```diff
@@ -78,5 +86,12 @@ public ContentResult OnGetValidateEpisodeProperty(string key, string value, bool
         {
             return validationService.ValidateModelProperty<Episode>(key, value, shouldValidateFull);
         }
+
+        public async Task<ContentResult> OnGetValidateNotificationDateAsync(string key, string day, string month, string year, int notificationId)

```

Comment:
We are doing something very similar on ClinicalDetails, where we compare BcgVaccinationYear to Dob - could we commonise? Check out the ValidateClinicalDetailsYearComparison method - I think that's just a less neat way of what you're doing here, and hopefully we can do the same in both cases (which would get rid of some methods on ValidationService!)

Reply:
This is a bit different as BcgVaccinationYear was compared to Year of dob, which was passed as parameter. Having the same for date of birth will require duplicate of all methods that BcgVaccinationYear uses but with datatime instead of int year. We can convert a year to datetime 1/1/year to compare if want to commonise this two comparisons. And I am sure if that will be neater.

Reply:
Ah yes sorry that's right, probably not worth changing then as it's not quite equivalent!